### PR TITLE
windows-nsis: add wintun support

### PR DESCRIPTION
**Pull request by lstipakov**
This adds "wintun" option to Windows installer,
which installs wintun driver and creates tun device.

Wintun installer is based on wintun's "msi-example" -
a wrapper over MSM - wintun's official distribution way.

Signed-off-by: Lev Stipakov <lev@openvpn.net>

### CODE REVIEW

**Comment by mattock** [comment]
This looks reasonable and Travis CI is happy. Two out of three Appveyor tests also pass. The failing Appveyor test seems like a red herring.

There is a small indentation issue in there - I'll mark it separately. Besides that I think this PR can be merged.

File: windows-nsis/openvpn.nsi [hunk 1]
```diff
@@ -436,6 +467,30 @@ Section /o "TAP Virtual Ethernet Adapter" SecTAP
 	WriteRegStr HKLM "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\${PACKAGE_NAME}" "tap" "installed"
 SectionEnd
 
+!ifdef TAPCTL_EXISTS
+Section "Wintun TUN driver (experimental)" SecWINTUN
+
+	SetOverwrite on
+
+    SetOutPath "$INSTDIR\bin"
```
Comment (mattock) [inline windows-nsis/openvpn.nsi]: Indentation off here.

Comment (lstipakov) [reply windows-nsis/openvpn.nsi]: fixed!

**Comment by lstipakov** [comment]
I think it makes sense to merge it after merging wintun patches: https://patchwork.openvpn.net/project/openvpn2/list/?series=545

I would also appreciate testing feedback, the installer could be found here: https://staging.openvpn.net/openvpn2/

**Comment by mattock** [comment]
@lstipakov if needed just ping me here once the patches are merged. This PR looks good.

**Comment by rozmansi** [comment]
@lstipakov, can you rebase this PR onto master now that we have VS2019 support in. I really hope to get my hands on this PR to review it sooner rather than later.

**Comment by lstipakov** [comment]
> @lstipakov, can you rebase this PR onto master now that we have VS2019 support in. I really hope to get my hands on this PR to review it sooner rather than later.

Rebased!

**Comment by lstipakov** [comment]
Since wintun support has been merged to master, I think this can me merged as well.

Ping @rozmansi @mattock

**Pull request merged**